samples/snippets/v3/alerts-client/snippets_test.py

```diff
@@ -123,18 +123,16 @@ def pochan():
 
 
 def test_list_alert_policies(capsys, pochan):
-    # Query snippets.list_alert_policies() for up to 5 seconds
+    # Query snippets.list_alert_policies() for up to 50 seconds
     # to allow the newly created policy to appear in the list.
     retry = 5
     while retry:
         snippets.list_alert_policies(pochan.project_name)
         out, _ = capsys.readouterr()
-        # Only check up to the first 20 characters of the display name
-        # as long strings printed to the console are truncated.
-        if pochan.alert_policy.display_name[0:20] in out:
+        if pochan.alert_policy.display_name in out:
```

Comment:
Is name truncation no longer a concern?

Reply:
I found an [open issue](https://github.com/pytest-dev/pytest/issues/3962) on pytest related to the assertion diff being truncated so I think this was the issue that I was seeing in the build logs.